test/libdnf/transaction/MergedTransactionTest.cpp

```diff
@@ -1,96 +1,79 @@
+#ifdef SKIP
+
 #include <set>
 #include <string>
 
-#include "../backports.hpp"
-
-#include "libdnf/hy-subject.h"
-#include "libdnf/nevra.hpp"
-#include "libdnf/transaction/RPMItem.hpp"
+//#include "libdnf/hy-subject.h"
+#include "libdnf/rpm/nevra.hpp"
+#include "libdnf/transaction/rpm_package.hpp"
 #include "libdnf/transaction/MergedTransaction.hpp"
-#include "libdnf/transaction/Transaction.hpp"
+#include "libdnf/transaction/transaction.hpp"
 #include "libdnf/transaction/Transformer.hpp"
 
 #include "MergedTransactionTest.hpp"
 
-using namespace libdnf;
+using namespace libdnf::transaction;
 
 CPPUNIT_TEST_SUITE_REGISTRATION(MergedTransactionTest);
 
 void
 MergedTransactionTest::setUp()
 {
-    conn = std::make_shared< SQLite3 >(":memory:");
-    Transformer::createDatabase(conn);
+    conn = new libdnf::utils::SQLite3(":memory:");
+    Transformer::createDatabase(*conn);
 }
 
-static libdnf::swdb_private::TransactionPtr
-initTransFirst(SQLite3Ptr conn)
+static TransactionPtr
+initTransFirst(libdnf::utils::SQLite3 & conn)
```

Comment:
What about `init_trans_first()`

Reply:
The whole file is disabled with #define SKIP macro.
The additional changes will take about a week and couple more commits.

Reply:
Then mark skip macro with TODO please with a list what have to be performed.